iot_verifier/src/region_cache.rs

```diff
@@ -0,0 +1,72 @@
+use crate::Settings;
+use helium_crypto::PublicKeyBinary;
+use helium_proto::Region as ProtoRegion;
+use iot_config_client::{
+    iot_config_client::{IotConfigClient, IotConfigClientError},
+    region_params_resolver::{RegionParamsInfo, RegionParamsResolver},
+};
+use retainer::Cache;
+use std::time::Duration;
+
+const CACHE_TTL: u64 = 86400;
+
+pub struct RegionCache {
+    pub iot_config_client: IotConfigClient,
+    pub cache: Cache<ProtoRegion, RegionParamsInfo>,
```

Comment:
can this just be `Region` if we don't need to distinguish it between the proto struct and a locally defined struct?